platforms/jvm/plugins-jvm-test-suite/src/main/java/org/gradle/api/plugins/JvmTestSuitePlugin.java

```diff
@@ -53,28 +52,6 @@ public void apply(Project project) {
         TestingExtension testing = project.getExtensions().getByType(TestingExtension.class);
         testing.getSuites().registerBinding(JvmTestSuite.class, DefaultJvmTestSuite.class);
 
-        project.getTasks().withType(Test.class).configureEach(test -> {
-            // The test task may have already been created but the test sourceSet may not exist yet.
-            // So defer looking up the java extension and sourceSet until the convention mapping is resolved.
-            // See https://github.com/gradle/gradle/issues/18622
-            test.getConventionMapping().map("testClassesDirs", () -> {
-                DeprecationLogger.deprecate("Relying on the convention for Test.testClassesDirs in custom Test tasks")
-                    .willBeRemovedInGradle9()
-                    .withUpgradeGuideSection(8, "test_task_default_classpath")
-                    .nagUser();
-
-                return ((JvmTestSuite) testing.getSuites().findByName(JvmTestSuitePlugin.DEFAULT_TEST_SUITE_NAME)).getSources().getOutput().getClassesDirs();
-            });
-            test.getConventionMapping().map("classpath", () -> {
-                DeprecationLogger.deprecate("Relying on the convention for Test.classpath in custom Test tasks")
-                    .willBeRemovedInGradle9()
-                    .withUpgradeGuideSection(8, "test_task_default_classpath")
-                    .nagUser();
-                return ((JvmTestSuite) testing.getSuites().findByName(JvmTestSuitePlugin.DEFAULT_TEST_SUITE_NAME)).getSources().getRuntimeClasspath();
-            });
-            test.getModularity().getInferModulePath().convention(java.getModularity().getInferModulePath());
```

Comment:
Did you intend to remove this modularity convention?

Reply:
bah

Reply:
sorry, PTAL again